src/lcm/argmax.py

```diff
@@ -1,31 +1,42 @@
 import jax
```

Comment:
```suggestion
from jax.ops import segment_max
```
I think we'll generally want to use that as a convention, there are only very few instances where `import jax` is used. 

As a convention, I'd suggest:
- leave `jnp`
- `jax.random` becomes `jrnd`
- everything else via `from jax.x import y`

Just a note: In `test_process_model`, we are using both `jnp` and `jax.numpy` explicitly -- any subtle reason for that or just local optimization?

Reply:
> For the `jax.random` module I believe its best to follow the examples from the `jax` documentation and do `from jax import random`. 

To the extent that this will never ever conflict with anything a user might do with `numpy.random`, fine with me!

> We do not need it that often currently anyways.

Other than `jax.numpy`, I think that was the most frequently used thing!